src/styles/app.scss

```diff
@@ -14,4 +14,30 @@ html, body, #root {
   display: block;
   height: 100%;
   width: 100%;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  background: #2E3133;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1rem;
+  text-rendering: geometricPrecision;
+}
+
+.main-loader {
+  background: #2E3133 url('/assets/img/paratii-loader.gif') no-repeat 50%;
+  background-size: cover;
+  height: 50px;
+  left: 50%;
+  position: fixed;
+  top: 50%;
+  transform: translate3d(-50%, -50%, 0);
+  width: 50px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: 400;
```

Comment:
I don't think this is going to work for embed. `embed/index.js` imports a separate `/embed/index.scss` file, not this one

Reply:
@bent0b0x, could we import all styles from `src/styles/app.scss`?

Reply:
You can update `embed/index.js`. Just note that we want to keep `/embed` light so if there things in here that the embed doesn't need let's not import them. Otherwise, import what is needed into `/embed/index.scss`

Reply:
Done https://github.com/Paratii-Video/paratii-portal/pull/324/commits/2800e1ddba828514e8a922e32a7618f0b3acab8e